validation/createCompanySchema.js

```diff
@@ -0,0 +1,35 @@
+import Joi from '@hapi/joi';
+
+export const createCompanySchema = Joi.object({
+  companyName: Joi.string()
+    .alphanum()
+    .min(3)
+    .max(30)
+    .required(),
+
+  acceptedTerms: Joi.bool().required(),
+  copyAcceptedTerms: Joi.bool().required(),
+  email: Joi.string()
+    .email()
+    .required(),
+  contactFirstName: Joi.string().required(),
+  companyLastName: Joi.string().required(),
+  iban: Joi.string().required(),
+  kvk: Joi.number()
+    .integer()
+    .min(10000000)
+    .max(99999999)
+    .required()
+    .messages({
+      'number.min': '"kvk" should be 8 digits long',
+      'number.max': '"kvk" should be 8 digits long'
+    }),
```

Comment:
Not beautiful, but this is how Joi validates numbers. I just had to override the message,